README.md

```diff
@@ -41,90 +43,157 @@ Add a `.stylelintrc` file to the root of your project:
 
-When using this processor a couple of stylelint rules throw errors that you cannot prevent. Like
-'[no-empty-source](https://stylelint.io/user-guide/rules/no-empty-source)' or
-'[no-missing-end-of-source-newline](https://stylelint.io/user-guide/rules/no-missing-end-of-source-newline)'.
+When using this processor a couple of stylelint rules throw errors that cannot be prevented, like [`no-empty-source`](https://stylelint.io/user-guide/rules/no-empty-source) or [`no-missing-end-of-source-newline`](https://stylelint.io/user-guide/rules/no-missing-end-of-source-newline). There's also a couple unnecessary ones like [`no-vendor-prefix` rules](https://stylelint.io/user-guide/rules/property-no-vendor-prefix). (`styled-components` automatically vendor prefixes your code, so you don't need to do it manually)
 
-The [stylelint-config-styled-components-processor](https://github.com/styled-components/stylelint-config-styled-components-processor)
-shareable config will automatically disable rules that cause unresolvable conflicts. Besides those
-rules vendor prefixed [properties](https://stylelint.io/user-guide/rules/property-no-vendor-prefix)
-and [values](https://stylelint.io/user-guide/rules/value-no-vendor-prefix) will throw an error since
-styled-components automatically generates vendor prefixes for your css. Note that if you want to
-change any of these rules you can always override them in your stylelint config.
+The [`stylelint-config-styled-components-processor`](https://github.com/styled-components/stylelint-config-styled-components-processor) will automatically disable rules that cause conflicts.
 
-### Webpack
+> **NOTE:** You can override rules defined in shared configs in your custom `.stylelintrc`.
 
-For use with Webpack you can use the [`stylelint-custom-processor-loader`](https://github.com/emilgoldsmith/stylelint-custom-processor-loader).
+### Interpolation tagging
 
-### Syntax notes
-#### Turning rules off from within your CSS
+Sometimes `stylelint` can throw an error (e.g. `CssSyntaxError`) even though nothing is wrong with your CSS. This is often due to an interpolation, more specifically the fact that the processor doesn't know what you're interpolating.
 
-Turning off rules with `stylelint-disable`-like comments (see the [stylelint documentation](https://stylelint.io/user-guide/configuration/#turning-rules-off-from-within-your-css) for all allowed syntax) is fully supported inside and outside of the tagged template literals, do note though that what actually happens behind the scene is that all `stylelint-(disable|enable)` comments are moved into the compiled css that is actually linted, so something like this:
+A simplified example:
 
+```js
+const something = 'background';
+
+const Button = styled.div`
+  ${something}: papayawhip;
+`
+```
+
+> **NOTE:** This example is just for illustration purposes, it would lint correctly. Imagine an imported utility from a third-party library, a global variable, etc.
```

Comment:
I believe this example would actually exactly throw an error, since we would guess it was a whole declaration as this is the start of a new declaration. Therefore with our substitution the CSS would probably end up looking like: 
```
  -styled-mixin0: dummyValue;: papayawhip
``` 
as far as I believe, so this note is incorrect, the example is good though :)

Reply:
Perfect, mind removing the note?

Reply:
Sure thing, I can update both of them